actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -148,6 +148,13 @@ private void updateUser(Request actorMessage) {
       userMap.put(JsonKey.UPDATED_BY, actorMessage.getContext().get(JsonKey.REQUESTED_BY));
     }
     Map<String, Object> requestMap = UserUtil.encryptUserData(userMap);
+    ProjectLogger.log("UserManagementActor:updateUser:user data successfully encrypted",LoggerEnum.INFO.name());
+    if(userMap.containsKey(JsonKey.RECOVERY_EMAIL)){
+      isRecoveryEmailEqualsPrimaryEmail(userDbRecord,(String) userMap.get(JsonKey.RECOVERY_EMAIL));
+    }
+    if(userMap.containsKey(JsonKey.RECOVERY_PHONE)){
+      isRecoveryPhoneEqualsPrimaryPhone(userDbRecord,(String) userMap.get(JsonKey.RECOVERY_PHONE));
```

Comment:
We can create a generic method as "isPrimaryIdentifierEqualsSecondary" , this will take primaryIdentifier attribute name, secondaryIdentifier attribute name.